simple/cm_data.c

```diff
@@ -191,6 +191,19 @@ static int server_accept(size_t paramlen)
 	}
 
 	fi_shutdown(ep, 0);
+	memset(entry, 0, sizeof(*entry) + paramlen);
+	ret = fi_eq_sread(eq, &event, entry, sizeof(*entry) + paramlen, -1, 0);
```

Comment:
nit: FI_SHUTDOWN event will not have cm-data. It is ok to have a larger buffer for eq_sread, but it will be better to avoid any confusion here.